util/src/main/java/com/google/cloud/hadoop/util/CredentialFactory.java

```diff
@@ -439,4 +458,12 @@ private HttpTransport getTransport() throws IOException {
   void setTransport(HttpTransport transport) {
     this.transport = transport;
   }
+
+  private static boolean isOAuthCredentialsConfigured(CredentialOptions options) {
```

Comment:
This seems so weird. Should we create bug in backlog to update this with `auth.type` token, same as in `master branch?